### ISSUE TITLE
Staging Review finding: Confusing image heading for screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#84226]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV-Medications
**Product:** MHV Medications
**Feature:** Medications

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

The medication detail pages (eg. /medications/prescription/[ID]) have a spot for images of the medication to appear --- but every medication I've checked with the test user has displayed an "Image not available" message. I'm interested in how you've implemented alt text for these images.

**Update:** Team provided an example medication with image and text description, so I was able to complete testing. Removing the `launch-blocking` label and updating the ticket with new notes.

As currently coded, the image is hidden from screen reader users using `aria-hidden`, but the H4 heading "Image" isn't. Screen reader users encountering that heading won't perceive any content following the heading, which may be interpreted as something being broken on the page or as though there's an image with useful information in it that's just coded inaccessibly.

## Recommended action

No recommendation except that it would be good to test! Flagging as a potential issue, can either share more or close once I get a look at how you've done this.

**Update:** I'd recommend adding `aria-hidden="true"` to the H4, or alternatively wrapping both the H4 heading and the image inside a `div` that has `aria-hidden="true"` on it. The heading serves no purpose besides helping to find the image, so if the image is hidden it can be hidden as well.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 1.1.1 Non-text Content - Level A](https://www.w3.org/TR/WCAG21/#non-text-content)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by kfink24** [comment]
@briandeconinck Medication name: SALICYLIC ACID 6% GEL 30 GM on page 4 has an image.

**Comment by briandeconinck** [comment]
Issue updated after testing the provided medication image. Thank you again for helping me check that so quickly!

**Comment by IdeaTrail** [comment]
Work will be done under Jira ticket: https://jira.devops.va.gov/browse/MHV-58774

**Comment by IdeaTrail** [comment]
Closing Github ticket. Tracking work under Jira ticket.

**Comment by shiragoodman** [comment]
Hi @IdeaTrail  - I am reopening this issue as we kindly request that Staging Review tickets not be closed until the issue has been resolved. Please let me know if you have any questions - ty!

**Comment by IdeaTrail** [comment]
Work completed under ticket https://jira.devops.va.gov/browse/MHV-58774
